testsuite/features/secondary/min_empty_system_profiles.feature

```diff
@@ -1,18 +1,16 @@
-# Copyright (c) 2018-2021 SUSE LLC
+# Copyright (c) 2018-2022 SUSE LLC
 # Licensed under the terms of the MIT license.
 
 @scope_onboarding
 Feature: Empty minion profile operations
 
-  Scenario: Create an empty minion profile with HW address via XML-RPC API
-    Given I am logged in via XML-RPC system as user "admin" and password "admin"
+  Scenario: Create an empty minion profile with HW address via API
+    Given I am logged in API as user "admin" and password "admin"
     When I call system.create_system_profile() with name "empty-profile" and HW address "00:11:22:33:44:55"
-    And I logout from XML-RPC system namespace
 
-  Scenario: Create an empty minion profile with hostname via XML-RPC API
-    Given I am logged in via XML-RPC system as user "admin" and password "admin"
+  Scenario: Create an empty minion profile with hostname via API
     When I call system.create_system_profile() with name "empty-profile-hostname" and hostname "min-retail.mgr.suse.de"
-    And I logout from XML-RPC system namespace
+    And I logout from API
```

Comment:
Same as for other cases, I think login/logout might be better in independent scenarios, as otherwise we have a login in one scenario that aimes to do something, and then logout in another kind of scenario.

Reply:
Yup. Like a "cleanup" step.

This is an example of where i moved from per-scenario to per-feature, but obviously it still needs some polishing.